Securonix - last run time fix (#5644)
demisto/content: Integrations/Securonix/CHANGELOG.md

## [Unreleased]

> Fixed an issue when the last time of the fetch was not updated correctly.

Reviewer: ```suggestion
Fixed an issue where the integration failed to fetch incidents.
```
